src/app/pages/send/send-crypto-asset-form/form/btc/use-btc-send-form.tsx

```diff
@@ -83,12 +83,12 @@ export function useBtcSendForm() {
     }),
 
     async chooseTransactionFee(
-      values: BitcoinSendFormValues,
-      formikHelpers: FormikHelpers<BitcoinSendFormValues>
+      values: BitcoinSendFormValues
+      // formikHelpers: FormikHelpers<BitcoinSendFormValues>
     ) {
       logger.debug('btc form values', values);
       // Validate and check high fee warning first
```

Comment:
I won't change this code yet but it seems like this validation never runs. 

We `await` but we don't assert that it succeeds so we just create a promise and then move ahead with the UI without confirming there is no validation errors